src/Connection.js

```diff
@@ -6,7 +6,16 @@ import { ControlLayer } from 'streamr-client-protocol'
 
 const debug = debugFactory('StreamrClient::Connection')
 
-class Connection extends EventEmitter {
+export default class Connection extends EventEmitter {
+    static get State() {
```

Comment:
Does this need to be a getter? Means every *access* to `Connection.State` instantiates a new `Object`

Reply:
You're right, I copied idea from Subscription.js for consistency, but it was wrong idea. Updated both now.